docs/manager/graphql-reference/schema.graphql

```diff
@@ -1975,7 +1975,12 @@ type Mutations {
   forget_image(architecture: String = "x86_64", reference: String!): ForgetImage @deprecated(reason: "Deprecated since 25.4.0. Use `forget_image_by_id` instead.")
 
   """Added in 25.4.0"""
-  purge_image_by_id(image_id: String!): PurgeImageById
+  purge_image_by_id(
+    image_id: String!
+
+    """Added in 25.5.0."""
+    options: PurgeImageByIdOptions = {remove_from_registry: false}
```

Comment:
Should a single type like `image_id` also be wrapped in a new Key type? @HyeockJinKim 
(If so, that PR should also be considered a breaking change.)